modules/cluster-logging-upgrading-logging-from-44.adoc

```diff
@@ -0,0 +1,306 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-upgrading.adoc
+
+[id="cluster-logging-updating-logging-from-44_{context}"]
+= Updating cluster logging from 4.4
+
+When you update the {product-title} cluster, you can update cluster logging from 4.4 to 4.6 by changing the subscription for the Cluster Logging Operator and the Elasticsearch Operator.
+
+When your cluster logging version is 4.4, you must update cluster logging as follows due to changes in the way that OLM provides operator packages.
+
+Additionally, it is recommended that you update Cluster Logging Operator before you update Elasticsearch Operator to avoid
+the need to restart each Elasticsearch pod manually, one by one.
+
+Perform each update, version by version, in the following order:
+
+. Update OpenShift Cluster to 4.5.
```

Comment:
We should upgrade Cluster Logging Operator and elasticsearch-operator to same version(4.5) on Openshift 4.5.
We should upgrade Cluster Logging Operator and elasticsearch-operator to same version(4.6) on Openshift 4.6. The order should as below

 Update OpenShift Cluster to 4.5. 
. Update Cluster Logging Operator to the latest version of 4.4, then update to 4.5.
.Update Elasticsearch Operator to the latest version of 4.4, then update to 4.5.
. Update OpenShift Cluster to 4.6.
. Update Cluster Logging Operator to latest version of 4.5, then update to 4.6
. Update Elasticsearch Operator to latest version of 4.5, then update to 4.6

Reply:
Done.

Reply:
Done.